genai-perf/genai_perf/metrics/telemetry_metrics.py

```diff
@@ -80,3 +80,8 @@ def __repr__(self):
     @property
     def telemetry_metrics(self) -> List[MetricMetadata]:
         return self.TELEMETRY_METRICS
+
+    @property
+    def data(self) -> dict:
```

Comment:
Will this get used anywhere?

Reply:
This will be used in TelemetryStatistics.